packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -505,28 +505,27 @@ class ComboBoxInternal extends React.Component<IComboBoxInternalProps, IComboBox
 
   /**
    * componentWillReceiveProps handler for the auto fill component
-   * Checks/updates the input value to set, if needed
+   * Checks/updates the input value to set, if needed.
    * @param defaultVisibleValue - the defaultVisibleValue that got passed
    *  in to the auto fill's componentWillReceiveProps
    * @returns - the updated value to set, if needed
    */
   private _onUpdateValueInAutofillWillReceiveProps = (): string | null => {
-    const comboBox = this._autofill.current;
+    if (this._autofill.current) {
```

Comment:
I don't think there was anything wrong with the way this code was before. Whether to bail early or wrap the whole thing in an if() is mainly a matter of preference (we don't have an official preference in this repo).